pkg/sync/sync.go

```diff
@@ -10,25 +10,25 @@ import (
 	dfutil "github.com/devfile/library/pkg/util"
 
 	"github.com/redhat-developer/odo/pkg/exec"
-	"github.com/redhat-developer/odo/pkg/kclient"
+	"github.com/redhat-developer/odo/pkg/platform"
 	"github.com/redhat-developer/odo/pkg/util"
 
 	"k8s.io/klog"
 )
 
 // SyncClient is a Kubernetes implementationn for sync
```

Comment:
```suggestion
// SyncClient is a Kubernetes implementation for sync
```

Or maybe something more appropriate.